applications/validator-setup-maintenance.md

```diff
@@ -0,0 +1,179 @@
+
+### Team Code Repos
+
+- https://github.com/Popular-Coding
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/Gryczka
+- https://github.com/GinSiuCheng
+
+### Team LinkedIn Profiles 
+
+- https://www.linkedin.com/in/gryczka/
+- https://www.linkedin.com/in/ginsiucheng/
+
+## Development Status :open_book:
+
+[**RFP** - Validator Setup Maintenance](https://github.com/w3f/Grants-Program/blob/master/rfps/validator-setup-maintenance.md)
+
+**Relevant Work**
+* [Azure AKS Deployment Repository](https://github.com/GinSiuCheng/private-aks-deployment) (Not Validator Specific)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 4-5 months 
+- **Full-Time Equivalent (FTE):** 2 FTEs
+- **Total Costs:** $50,000 USD
+### Milestone 1 Terraform Remote Backend and Metrics-based notification implementation
+
+- **Estimated duration:** 1 Month
+- **FTE:**  2
+- **Costs:** $10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation for remote backend configurations | We will provide inline documentation of the code and a basic tutorial on configuring remote backends for terraform leveraging Azure Blob Storage, Google Cloud Storage and AWS S3 Storage. |
+| 0c. | Inventory list of metric-based alerts for AWS, Azure and GCP | We will provide an inventory list of metrics based alerts for Azure, AWS and GCP that should be configured based on the infrastructure deployed to notify end users of operational downtimes and errors. |
+| 1. | Terraform remote backend implementation | We will provide implementation examples for Azure, AWS and GCP storage for Terraform remote backend that includes CLI commands to instantiate the cloud storage needed, terraform configurations for the remote backends and protection mechanisms (resource-lock) to prevent accidental deletion. |  
+| 2. | Metrics-based alerts and notifications | We will create metrics-based alerts and notifications in terraform for Azure, AWS and GCP based on the inventory list provided, metrics available and terraform cloud provider support. |  
+
+### Milestone 2 — In Repo Deploy to Cloud Buttons - AWS & Azure
+
+- **Estimated Duration:** 2 months
+- **FTE:**  2
+- **Costs:** $15,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Overview of cloud deployment button - Documented through a readme page | Provide an overview documentation on code structure for AWS/Azure cloud deploy button |
+| 1a. | Documentation on AWS deployment button | We will provide inline documentation on the deployment button for AWS | 
+| 1b. | AWS deployment button for validator | Rewrite existing assets/scripts to match cloudformation deployment button model for AWS deploy | 
```

Comment:
Deployment buttons are tied to cloud specific IaC options, like CloudFormation Templates for AWS and ARM Templates for Azure.  This would add more IaC to maintain.

The trade off is that it would also provide an engaging and quickly accessible means of deployment for any existing AWS and Azure users interacting with the repository.

The additional IaC options could also be of value for any organizations which have not committed to Terraform as a deployment strategy.

Long term, the maintenance requirements for these templates would be minimal.  Updates would largely be tied to updating instance selections, which should coincide directly with respective updates to each provider’s Terraform files.

AWS Deployment Buttons
https://aws.amazon.com/blogs/devops/construct-your-own-launch-stack-url/
Azure Deployment Buttons
https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-to-azure-button